### ISSUE TITLE
Device Request

### ISSUE THREAD

**Issue by lovozeto** [bwp91/homebridge-govee#827]
* **Which model is this? Need the `Hxxxx` number.**
H605A

* **Which type of device is this? Light, heater, leak sensor, etc.**
TV Backlight Kit

* **Which connection methods does this device have? Wifi? Bluetooth? LAN mode? All?**
All

* **Anything else?**
*[7/3/2024, 12:57:02 PM] [Govee] [BLE] disabling client as ENODEV, No such device.
[7/3/2024, 12:57:02 PM] [Govee] [TV Backlight Kit] is not currently supported, do create a Github issue with the following info:
{"device":"60:EB:C9:F0:81:06:44:96","deviceName":"TV Backlight Kit","model":"H605A","httpInfo":{"groupId":0,"device":"60:EB:C9:F0:81:06:44:96","sku":"H605A","spec":"","versionHard":"3.05.01","versionSoft":"1.00.05","deviceName":"TV Backlight Kit","deviceExt":{"deviceSettings":"{\"wifiName\":\"XXXX\",\"address\":\"C9:F0:81:06:44:96\",\"bleName\":\"Govee_H605A_4496\",\"topic\":\"GD/6c05387be744b43eb8f32a0e7aa4d0a2\",\"wifiMac\":\"60:74:F4:E9:94:DD\",\"pactType\":1,\"pactCode\":1,\"wifiSoftVersion\":\"1.00.14\",\"wifiHardVersion\":\"1.06.00\",\"ic\":56,\"secretCode\":\"+FZ0TTy7+Ss=\",\"boilWaterCompletedNotiOnOff\":1,\"completionNotiOnOff\":1,\"autoShutDownOnOff\":1,\"sku\":\"H605A\",\"device\":\"60:EB:C9:F0:81:06:44:96\",\"deviceName\":\"TV Backlight Kit\",\"versionHard\":\"3.05.01\",\"versionSoft\":\"1.00.05\",\"playState\":false}","lastDeviceData":"{\"online\":false}","extResources":"{\"skuUrl\":\"https://d1f2504ijhdyjw.cloudfront.net/sku-img/b0fb924ba87ea0c3950888c3ec1bc42b-add_list_type_device_605a.png\",\"headOnImg\":\"\",\"headOnImgNew\":\"https://d1f2504ijhdyjw.cloudfront.net/sku-img/44465ea19aacbbd800a158e4058ca489-new_light_title_605a_on.png\",\"headOffImg\":\"\",\"headOffImgNew\":\"https://d1f2504ijhdyjw.cloudfront.net/sku-img/9cdc37821a2cb09e7197e23c8ffb28e8-new_light_title_605a_off.png\",\"ext\":\"\",\"ic\":56}"},"pactType":1,"pactCode":1,"share":0,"deviceId":22004186,"gidConfirmed":false,"goodsType":253,"attributesId":33,"supportScene":0}}

**Comment by benwebbbenwebb** [comment]
Hi @lovozeto 

FYI looks like it should be supported now

https://github.com/bwp91/homebridge-govee/blob/fa245b4e0e04a325f74aea6837128dfa9d5acc7d/CHANGELOG.md?plain=1#L30-L38

**Comment by bwp91** [comment]
@lovozeto are you sure this device supports LAN mode?

**Comment by lovozeto** [comment]
Yes it does, with the latest update it got recognised and added to homebridge:

> `[22/07/2024, 13:23:40] [Govee] [LAN] client enabled and found 0 device(s).
> [22/07/2024, 13:23:41] [Govee] [HTTP] client enabled and found 2 device(s).
> [22/07/2024, 13:23:41] [Govee] [AWS] client enabled.
> [22/07/2024, 13:23:41] [Govee] [BLE] client enabled.
> [22/07/2024, 13:23:41] [Govee] [Floor Lamp Basic] initialising with options {"adaptiveLightingShift":0,"aws":"enabled","ble":"enabled","brightnessStep":1,"colourSafeMode":false,"lan":"unsupported"}.
> [22/07/2024, 13:23:41] [Govee] [Floor Lamp Basic] initialised with id [28:D0:D1:35:34:39:1C:2F] [H6076].
> [22/07/2024, 13:23:41] [Govee] [TV Backlight Kit] initialising with options {"adaptiveLightingShift":0,"aws":"enabled","ble":"enabled","brightnessStep":1,"colourSafeMode":false,"lan":"unsupported"}.
> [22/07/2024, 13:23:41] [Govee] [TV Backlight Kit] initialised with id [60:EB:C9:F0:81:06:44:96] [H605A].
> [22/07/2024, 13:23:41] [Govee] [TV Backlight Kit] not using AWS connection as not connected to AWS.
> [22/07/2024, 13:23:42] [Govee] ✓ Setup complete. This plugin has been made with ♥ by bwp91, please consider a ☆ on GitHub if you are finding it useful!
> [22/07/2024, 13:23:44] [Govee] [Floor Lamp Basic] current brightness [15%].
> [22/07/2024, 13:23:44] [Govee] [Floor Lamp Basic] current colour [3800K / 263M].
> [22/07/2024, 13:23:44] [Govee] [TV Backlight Kit] current brightness [39%].
> [22/07/2024, 13:23:44] [Govee] [TV Backlight Kit] current colour [5000K / 200M].
> [22/07/2024, 13:23:46] [Govee] [TV Backlight Kit] current colour [3500K / 284M] via adaptive lighting.
> [22/07/2024, 13:23:54] [Govee] [TV Backlight Kit] could not be updated as Promise timed out after 10000 milliseconds at file:///var/lib/homebridge/node_modules/homebridge-govee/node_modules/p-timeout/index.js:70:24.
> [22/07/2024, 13:23:54] [Govee] [TV Backlight Kit] could not find device [c9:f0:81:06:44:96].`

Although there's an error, the lights appear en in the Home App and I can control the lights.
